Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16-alpine
+
+RUN apk add python3 make gcc musl-dev pkgconfig libusb-dev g++ linux-headers eudev-dev
```

Comment:
What are we compiling?

Reply:
Should be removed altogether

Reply:
all of these are needed for npm install of all breadcrumbs dependencies. the container does not build if you remove any of these packages.

Reply:
ok, are you going to move it to optional?